bears/general/CPDBear.py

```diff
@@ -0,0 +1,41 @@
+import re
+import os
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+
+
+class CPDBear(LocalBear, Lint):
```

Comment:
oh yes this should certainly be a global bear and detect duplicates across files! (@hypothesist challenging example for caching :))